src/test/shared/sam/cli/samCliDeploy.test.ts

```diff
@@ -68,7 +69,8 @@ describe('SamCliDeployInvocation', async () => {
                 ['key1', 'value1'],
                 ['key2', 'value2'],
             ]),
-            invoker
+            invoker,
+            'profile'
```

Comment:
No additional testing needed at this level considering this is a mandatory parameter. We will want to add some testing when we add integration tests.

Reply:
samCliLocalInvoke.test.ts contains unit tests that verify we pass expected values to the invoker.invoke call.
Add a similar one for sam cli deploy to test profile. Tests can be backfilled for existing untested code later, but incremental changes that can be tested should be tested.

Reply:
✔️